src/main/webapp/timestamps.js

```diff
@@ -0,0 +1,39 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Coverts a Date representing a video {@code timestamp} into a string.
+ */
+function timestampToString(timestamp) {
+  const ts = new Date(timestamp);
```

Comment:
Please avoid abbreviations, I have mentioned the naming best practices. Please look at those: https://docs.google.com/document/d/1_ldClEZLsI1cMpQl_BfeIlkjTsE2IiJ2PgvhABfO6vs/edit#

Seems a better name is date because that is what it is.

Reply:
Fixed, thanks